test/web-platform-tests/to-run.yaml

```diff
@@ -28,7 +28,7 @@ basic.htm: [fail, Unknown]
 client-hint-request-headers-2.tentative.htm: [fail, Client Hints not implemented]
 client-hint-request-headers.htm: [fail-slow, Client Hints not implemented]
 cors-safelisted-request-header.any.html: [fail, Depends on fetch]
-credentials-flag.htm: [fail, Unknown]
+credentials-flag.htm: [needs-node12, Unknown]
```

Comment:
Weird.

Reply:
Yep, presumably a fix from the new HTTP parser.